api/envoy/api/v2/route/route.proto

```diff
@@ -587,7 +607,25 @@ message RedirectAction {
     //
     // .. note::
     //
-    //   As an example, given following config,
+    //   Take a careful attention to the use of trailing slash in match prefix value. A common
+    //   use-case is to have a rule to rewrite e.g. :code:`scheme://host/prefix` and
+    //   :code:`scheme://host/prefix/ok/1/2/3` to the root path of the host (:code:`scheme://host/`)
+    //   and :code:`scheme://host/ok/1/2/3` respectively.
+    //
+    //   The following config example exhibits that behavior.
+    //
+    //   .. code-block:: yaml
+    //
+    //     - match:
+    //         prefix: "/prefix/"
+    //       redirect:
+    //         prefix_rewrite: "/"
+    //     - match:
+    //         prefix: "/prefix"
+    //       redirect:
+    //         prefix_rewrite: "/"
```

Comment:
I actually don't have the whole context for this file, but I wonder if you could point to the first instance of this example rather than repeating it.

Reply:
Let me see